src/include/CMakeLists.txt

```diff
@@ -55,6 +55,22 @@ configure_file (
 	"${CMAKE_CURRENT_BINARY_DIR}/kdbversion.h"
 	)
 
+file (MAKE_DIRECTORY ${CMAKE_BINARY_DIR}include/)
+configure_file (
```

Comment:
The reason was that i did not know which would be called first. I cleaned it up now. I think this still needs a cleaner solution in general, but that is out of the scope of the pull request.